bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,48 @@
+# speedtest Binding
+
+This binding simplifies the addition of network speed tests from your openHAB instance.   Simply put, it's a wrapper around Ookla's SpeedTest ( https://www.speedtest.net/apps/cli ).  
+
+You MUST have the Ookla cli application installed on your machine prior to using this.
+
+## Supported Things
+
+Speedtest thing.   
+
+## Binding Configuration
+
+For this binding to work, you MUST install Ookla's speedtest, this will not work with other versions of "speedtest-cli" or packaged speedtest varieties.   
+
+To install Ookla's version of speedtest, head to https://www.speedtest.net/apps/cli and follow the instructions for your Operating System.   
```

Comment:
Please change every mention of Speedtest to the correct spelling, unless it is a path name or a web address where it needs to be all lower case.

Reply:
done